templates/CRM/Event/Page/EventInfo.tpl

```diff
@@ -75,7 +75,7 @@
 <div class="vevent crm-event-id-{$event.id} crm-block crm-event-info-form-block">
   <div class="event-info">
   {* Display top buttons only if the page is long enough to merit duplicate buttons *}
-  {if $event.summary or $event.description}
+  {if array_key_exists('summary', $event) or array_key_exists('description', $event)}
```

Comment:
If event.summary and event.description exist but are both blank this would still show this section.

Most of the ones below too - it's no longer checking that there is a non-blank value.

Reply:
yeah - it seems to filter them out if blank but I guess that could change in the future ? I thought it would be easier to read shorter but I can do both